### PR TITLE
Changes for domain creation in zephyr-dom0-xt

### PR DESCRIPTION
**Pull request by firscity**
This PR substitutes domd binary with Zephyr OS DomU for testing purposes.

Also it removes domain auto-start on boot, that is not needed in this project. Auto-start change should be squashed during migration to main branch, domain can still be created with `xu create -d 1`

### CODE REVIEW

File: src/domd_cfg.c [hunk 1]
```diff
@@ -377,21 +377,21 @@ static uint32_t domd_irqs[] = {
...
-	.flags = (XEN_DOMCTL_CDF_hvm | XEN_DOMCTL_CDF_hap | XEN_DOMCTL_CDF_iommu),
+	.flags = (XEN_DOMCTL_CDF_hvm | XEN_DOMCTL_CDF_hap),
 	.max_evtchns = 10,
 	.max_vcpus = 4,
 	.gnt_frames = 32,
 	.max_maptrack_frames = 1,
 
 	.iomems = domd_iomems,
-	.nr_iomems = sizeof(domd_iomems) / sizeof(*domd_iomems),
+	.nr_iomems = 0 /* sizeof(domd_iomems) / sizeof(*domd_iomems) */,
```
Comment (oleksiimoisieiev) [inline src/domd_cfg.c:387]: Please remove comments. And why are you changing domd config? Accoding to the commit message - you are fixing domu

Comment (firscity) [reply src/domd_cfg.c:387]: This commit has been done only for demonstration purposes - to boot any domain under Qemu. These changes are not about to left in main branch. Final goal is to boot DomD on some HW, where all of these structs will be filled with proper values.

Comment (oleksiimoisieiev) [reply src/domd_cfg.c:387]: I see. Then I'd rather change name to WIP and leave a comment that those changes should be used for the future implementations.

Comment (firscity) [reply src/domd_cfg.c:387]: @oleksiimoisieiev this PR will be closed and commit moved to xentroops WIP branches, thanks for review

Comment (oleksiimoisieiev) [reply src/domd_cfg.c:387]: ok

**Pull request closed without merging**